client/webpack.client.base.config.js

```diff
@@ -20,9 +21,7 @@ module.exports = {
     ],
 
     // This will contain the app entry points defined by webpack.hot.config and webpack.rails.config
-    app: [
-      './app/bundles/comments/startup/clientRegistration',
-    ],
+    app: glob.sync('./app/bundles/**/startup/clientRegistration.js*'),
```

Comment:
It's ok if we want to compile all bundles into single output, but usually it looks like this:

```js
someBundle: './app/bundles/someBundle/startup/clientRegistration.js',
someOtherBundle: './app/bundles/someOtherBundle/startup/clientRegistration.js',
```

So globbing doesn't help much.